pegasus/sites.v3/translate.hourofcode.com/i18n/en.yml

```diff
@@ -0,0 +1,216 @@
+en-US:
```

Comment:
All of the files with strings in them are sent to crowdin. When I build the project and download the translations, crowdin sets up a pseudo-language (in this case en-US) where it replaces all the strings with codes. So for the markdown files, it's easily replaced and I just added the script at the top. But with index.haml and the views, where we use hoc_s... it's not generated. The en.yml file is generated, so I had to replace them in the views. It was easy enough because there weren't that many and the site is very static. But this is certainly something to consider.